.github/workflows/runTests.yml

```diff
@@ -33,9 +33,7 @@ jobs:
       run:  tar --exclude="**/-lang:c++.zip" --exclude=".git*/" --exclude="htmlcov/" -acvf ../${{ runner.os }}.zip *
     - name: Create build archive (Linux)
       if: runner.os == 'Linux'
-      run: |
-        rm -rf ../${{ runner.os }}.zip
-        zip -r ../${{ runner.os }}.zip * -x "**/-lang:c++.zip" ".git*/" "htmlcov/"
+      run: zip -r ../${{ runner.os }}.zip * -x "**/-lang:c++.zip" ".git*/" "htmlcov/*"
```

Comment:
Does this resolve the merge rather than replace issue?

Reply:
No, the "remove build archive" step does